src/v/features/feature_table.h

```diff
@@ -60,7 +60,6 @@ enum class feature : std::uint64_t {
     force_partition_reconfiguration = 1ULL << 26U,
     raft_append_entries_serde = 1ULL << 28U,
     delete_records = 1ULL << 29U,
-    lightweight_heartbeats = 1ULL << 30U,
```

Comment:
is this really "dead" in the sense that this value could be contained in a controller log somewhere and need to be replayed?

Reply:
Its a retired flag as it is enabled by default in any cluster installing this version. More details here
https://github.com/redpanda-data/redpanda/blob/dev/src/v/features/feature_table.h#L88

Reply:
oh cool i didn't know about that. thanks